src/api.rs

```diff
@@ -84,13 +84,14 @@ pub struct EncoderConfig {
   pub min_quantizer: u8,
   pub bitrate: i32,
   pub tune: Tune,
-  /// log2(tile columns). Unused if tiles is specified.
-  pub tile_cols_log2: usize,
-  /// log2(tile rows). Unused if tiles is specified.
-  pub tile_rows_log2: usize,
-  /// Number of tiles desired. The video is split automatically so that
-  /// it contains at least this many tiles. Overrides tile_cols_log2
-  /// and tile_rows_log2.
+  /// Number of tiles horizontally. Encoder may round this number up
+  /// due to bitstream limitations.
+  pub tile_cols: usize,
+  /// Number of tiles vertically. Encoder may round this number up
+  /// due to bitstream limitations.
+  pub tile_rows: usize,
+  /// Total number of tiles desired. Encoder will try to optimally split
+  /// to reach this number of tiles, rounded up.
```

Comment:
Needs to document which takes precedence if both are set.

Reply:
Do you have a preference for the precedence?

Reply:
Nope... whichever floats your boat.

Reply:
Added.